update/provisioner.go

```diff
@@ -52,6 +52,9 @@ type Config struct {
 	Username string `mapstructure:"username"`
 	Password string `mapstructure:"password"`
 
+	// Adds a limit to how many updates are installed at a time
+	UpdateLimit string `mapstructure:"update_limit"`
```

Comment:
Because semantically the `UpdateLimit` is an `int` and not a `string`. The need to convert it to a string is a specific requirement for the command execution interface.